src/components/RouterComponent/RouterComponent.tsx

```diff
@@ -17,6 +23,9 @@ function RouterComponent() {
         <Route path="/cardetails/:id" element={<CarDetails />} />
         <Route path="/booking/:id" element={<Booking />} />
         <Route path="/booking_confirmation" element={<Confirmation />} />
+        <Route path="/careers" element={<Careers />} />
+        <Route path="/contact" element={<Contact />} />
+        <Route path="/testdrive" element={<TestDrive />} />
         <Route path="/page_notfound" element={<NotFound />} />
         <Route path="*" element={<Navigate to="/page_notfound" />} />
```

Comment:
Now that we are changing routes I would also like to see, a single pattern being used for the routes
lets follow the snake_case for routes, so 

- cardetails becomes `car_details`
- testdrive becomes `test_drive`


Also we do not need 2 routes here
we could simply have
`<Route path="*" element={<NotFound />} />`

Reply:
@pashedin did these routing changes